source/cart.html

```diff
@@ -7,7 +7,7 @@ <h1 class="page-title has-dash">{{ page.name }}</h1>
         {% for item in cart.items %}
           <li class="cart-item" data-item-id="{{ item.id }}">
             <a class="cart-item-image" href="{{ item.product.url }}">
-              <img src="{{ item.product.image | product_image_url | constrain: 200, 200 }}" alt="View {{ item.product.name | escape }}" title="View {{ item.product.name | escape }}" />
+              <img src="{{ item.product.image | product_image_url | constrain: 200, 200 }}" alt="View {{ item.product.name | escape }}" />
             </a>
             <a class="cart-item-details" href="{{ item.product.url }}" title="View {{ item.product.name | escape }}">

```

Comment:
WAVE didn't mark this title attr with a warning, and I wasn't sure about removing it in this context. The title is, in some ways, more descriptive than the content of the link, which would include all this info, for example:

<img width="218" alt="Screen Shot 2019-12-17 at 1 29 07 PM" src="https://user-images.githubusercontent.com/13512356/71023667-381e8780-20d1-11ea-895a-ca6c50f1987e.png">

Thoughts?

Reply:
@nickendle ^

Reply:
I think I'm fine with leaving the "View ..." title in there, for the reason you mentioned!